src/runtime-rs/crates/runtimes/common/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "common"
+version = "0.1.0"
+authors = ["The Kata Containers community <kata-dev@lists.katacontainers.io>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "^1.0"
+async-trait = "0.1.48"
+containerd-shim-protos = { version = "0.2.0", features = ["async"]}
+lazy_static = "1.4.0"
+nix = "0.16.0"
```

Comment:
This is really old. Any reason to stick on such an old version?

Reply:
Thanks for your review, already update nix to the latest version.